config/standalone.example.toml

```diff
@@ -130,3 +130,5 @@ retry_delay = "500ms"
 dir = "/tmp/greptimedb/logs"
 # Specify the log level [info | debug | error | warn]
 level = "debug"
+# Set the binding address for tokio-console subscriber.
+tokio_console_addr = "127.0.0.1:6669"
```

Comment:
I suggest not to include this option in user-facing sample configuration, otherwise we will need to explain what tokio and its console is.

Reply:
The tokio_console_addr is now set as a command line argument, PTAL